src/puppetlabs/trapperkeeper/services.clj

```diff
@@ -22,6 +22,8 @@
   (service-context [this] "Returns the context map for this service")
   (get-service [this service-id] "Returns the service with the given service id")
   (get-services [this] "Returns a sequence containing all of the services in the app")
+  (call-service-fn [this other-service-id fn-name args default] "Calls named function in other service, returning default if that service is not included.")
+  (service-included? [this service-id] "Returns nil if service does not exist and the service if it does")
```

Comment:
Yup, that's what `get-service` does.

In an ideal world I think it would be better for `get-service` to not throw in that case.  But, given that it's all shipped as such, this is probably reasonable.